### PR TITLE
added examples for transforms. [skip ci]

### PR DESCRIPTION
**Pull request by nitin42**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | no
| Major: Breaking Change?  | no 
| Minor: New Feature?      | no
| Deprecations?            | no
| Spec Compliancy?         | no
| Tests Added/Pass?        | no
| Fixed Tickets            | `Fixes #1, Fixes #2` 
| License                  | MIT
| Doc PR                   | yes 
| Dependency Changes       | no

Added examples for the remaining plugins !

### CODE REVIEW

File: packages/babel-plugin-transform-es2015-block-scoping/README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+**In**
+```javascript
+let msg = "Hello World!";
+```
+
+**Out**
+```javascript
+var msg = "Hello World!";
+```
+
```
Comment (xtuc) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: I think it would be interessting to demonstrate `throwIfClosureRequired` #5246

**Comment by nitin42** [comment]
Sorry! I made an error while adding the skip ci.

File: packages/babel-plugin-transform-es2015-block-scoping/README.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 > Compile ES2015 block scoping (const and let) to ES5
 
+## Examples
+
+**In**
+```javascript
+let msg = "Hello World!";
```
Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: Closures ! Right?

File: packages/babel-plugin-transform-es2015-block-scoping/README.md [hunk 3]
```diff
@@ -2,6 +2,32 @@
...
+var _loop = function _loop(i) {
+    setTimeout(function logger() {
+        console.log(i);
+    }, i * 1000);
+};
+
+for (var i = 1; i <= 10; i++) {
+    _loop(i);
+}
+```
```
Comment (xtuc) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: Could we also add a more simple example? Like:

in

```js
{
  let a = 3
}

let a = 3
```

out

``` js
{
  var _a = 3;
}

var a = 3;
```

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: Why not the current one ? Something ?

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: I know its not casual !

Comment (xtuc) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: The current one is good but to complex IMO. My example above shows the basic transformation.

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: So should I replace it with this example ?

Comment (xtuc) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: I think we should add theses two examples in the documentation.

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: Ok. What about the rest ?

Comment (xtuc) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: What do you mean with the rest? The rest of your PR?

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: Yes. The remaining examples for the plugins.

Comment (xtuc) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: The rest looks good to me

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: Gr8!

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-block-scoping/README.md]: had to merge first!

**Comment by aaronang** [comment]
In the future, I would recommend you to work in a separate branch other than `7.0`. This allows you to keep you `7.0` branch up-to-date with the `upstream` (`babel:7.0`) while not affecting your pull requests where you compare your `nitin42:feature-branch` with `babel:7.0`.

**Comment by nitin42** [comment]
Ok ! 😄

**Comment by xtuc** [comment]
Looks fine for me. Could you please rebase with 7.0?

**Comment by nitin42** [comment]
there were some merge conflicts and the commits were not squashed.

**Pull request merged**